lib/SizeFormatHelpers.js

```diff
@@ -7,6 +7,10 @@
 const SizeFormatHelpers = exports;
 
 SizeFormatHelpers.formatSize = size => {
+	if (typeof size !== "number" || Number.isNaN(size) === true) {
```

Comment:
` === true` is redundant.

Reply:
What do you mean?

That's a more explicit check otherwise the engine has to run the `ToBoolean` coercion.

Reply:
`Number.isNaN` already returns a boolean. There is no benefit, performance-wise nor readability-wise (granted, readability is subjective), to add `=== true` behind.

Reply:
I would leave comparison there, but after searching codebase there is someplaces where it is not used: https://github.com/webpack/webpack/blob/fde018300aa52262c384e937c408d5dd97d62951/lib/DllPlugin.js#L26

Reply:
To me, `if (Number.isNaN(size))` mentally parses as "if "size" is NaN then" and `if (Number.isNaN(size) === true)` parses as "if ("size" is NaN) equals true then". That's why the former is more readable. [Airbnb styleguide seems to agree with that](https://github.com/airbnb/javascript#comparison--shortcuts), couldn't find any sources that defends the latter.
Also did a little performance benchmark and can confirm that the former is faster.
It's ofcourse only a minor issue and in part a matter of personal preference.